doc-content/drools-docs/src/main/asciidoc/DecisionEngine/performance-tuning-decision-engine-ref.adoc

```diff
@@ -36,3 +36,18 @@ try {
 
 For information about built-in event listeners and debug logging in the {DECISION_ENGINE}, see xref:engine-event-listeners-con_decision-engine[].
 --
+
+Use lambda externalization for executable models::
+Enable the lambda externalization which optimizes the memory consumption at runtime. You can rewrite the lambdas that are generated and used in an executable model, enabling you to reuse the same lambda multiple times with all the patterns and the same constraint. Once the rete or phreak instantiates from an executable model, it becomes garbage collectible.
```

Comment:
This is something that happens under the hood, so it won't be "you" rewriting the lambda but the compilers. I'd say `It rewrites the generated lambdas used in the executable model...` (the executable model is always singular)


Also "once the rete is instantiated (passive form) from the executable model, it becomes garbage collectible (it refers to the exec model).

Reply:
Done